geoapi/src/main/python/opengis/metadata/base.py

```diff
@@ -18,163 +58,212 @@ class PortrayalCatalogueReference(ABC):
+        Citation for the standards to which the metadata conforms.
+
+        NOTE: Metadata standard citations should include an identifier.
+        """
 
     @property
-    def metadata_profile(self) -> Sequence[Citation]:
-        """Citation(s) for the profile(s) of the metadata standard to which the metadata conform."""
-        return None
+    @abstractmethod
+    def metadata_profile(self) -> Optional[Sequence[Citation]]:
+        """
+        Citation(s) for the profile(s) of the metadata standard to which the
+        metadata conform.
+
+        NOTE: Metadata citations should include an identifier.
+        """
+
+    @property
+    @abstractmethod
+    def alternative_metadata_reference(self) -> Optional[Sequence[Citation]]:
+        """
+        Reference to alternative metadata,e.g., Dublin Core, FGDC, or metadata
+        in a non-ISO standard for the same resource.
+        """
 
     @property
-    def alternative_metadata_reference(self) -> Sequence[Citation]:
-        """Unique Identifier and onlineResource for alternative metadata."""
-        return None
+    @abstractmethod
+    def other_locale(self) -> Optional[Sequence[str]]:
+        """
+        Provides information about an alternatively used localized character
+        strings. A string conforming to IETF BCP 47.
+        """
 
     @property
-    def metadata_linkage(self) -> Sequence[OnlineResource]:
+    @abstractmethod
+    def metadata_linkage(self) -> Optional[Sequence[OnlineResource]]:
         """Online location where the metadata is available."""
-        return None
 
     @property
-    def spatial_representation_info(self) -> Sequence[SpatialRepresentation]:
+    @abstractmethod
+    def spatial_representation_info(self) -> Optional[Sequence[
+        SpatialRepresentation
+    ]]:
         """Digital representation of spatial information in the dataset."""
-        return None
 
     @property
-    def reference_system_info(self) -> Sequence[ReferenceSystem]:
+    @abstractmethod
+    def reference_system_info(self) -> Optional[Sequence[ReferenceSystem]]:
         """
-        Description of the spatial and temporal reference systems used in the dataset.
-        The reference system may be:
-
-        * An ISO 19111 object such as ``CoordinateReferenceSystem``.
-        * A ``ReferenceSystem`` with the ``identifier`` property (from ISO 19111) sets to a list of ``Identifier`` values such as ``["EPSG::4326"]``.
-        * An object with the ``referenceSystemIdentifier`` property (from ISO 19115) sets to a single ``Identifier`` value such as ``"EPSG::4326"``,
-          optionally with a ``referenceSystemType`` property sets to a value such as ``geodeticGeographic2D`` or ``compoundProjectedTemporal``.
+        Description of the spatial and temporal reference systems used in
+        the dataset.
 
-        :rtype: Sequence[ReferenceSystem]
+        The reference system may be:
+        * An ISO 19111 object such as `CoordinateReferenceSystem`.
```

Comment:
In `CurveRelativeDirection(Enum)`, the item list was indented and used the `-` bullet instead of `*`. Maybe we should apply a uniform formatting? Both are fine for me, I just suggest to pickup one and use it for every bullet lists.